etc/deploy/tracing/jaeger-all-in-one.yaml

```diff
@@ -1,14 +1,19 @@
 # copied from
 # https://github.com/jaegertracing/jaeger-kubernetes/blob/bcca10c297b38c01d9d7c554172572ffa11944ce/all-in-one/jaeger-all-in-one-template.yml
```

Comment:
Should probably upgrade to the latest version of this manifest.

Reply:
Ah, I'll change this comment, but I did pull the latest as part of creating this PR and nothing had changed!

Reply:
Oh excellent.  You can submit this without waiting for tests, I don't think the tests test this.